source/local/local.go

```diff
@@ -133,6 +168,8 @@ func parseFeatures(lines [][]byte) map[string]string {
 
 // Run all hooks and get features
 func getFeaturesFromHooks() (map[string]string, error) {
+	klog.Warning("hooks are DEPRECATED since v0.12.0; use feature files instead")
```

Comment:
Yeah, it will be v0.12.0 so it's correct. We can keep it here or drop it whatever you like.

Unrelated: maybe you could supplement this with a mention that they will be dropped
```suggestion
	klog.Warning("hooks are DEPRECATED since v0.12.0 and support will be removed in a future release; use feature files instead")
```

Reply:
Thanks, updated.